test/tests/functional/pbs_qsub.py

```diff
@@ -41,6 +41,10 @@
 
```

Comment:
It is usually seen that new test cases are added in separate files. We do have a pbs_qstat.py, which should have been used for adding the testsuite/case in pbs_qstat_tag.py, but that was not done. 
So, I am not sure, if we can have so generic file names and contributors will choose to add their test suites or test cases to that file.